tests/requirements-py3.txt

```diff
@@ -13,7 +12,7 @@ pytest-twisted >= 1.11
 pytest-xdist
 sybil >= 1.3.0  # https://github.com/cjw296/sybil/issues/20#issuecomment-605433422
 testfixtures
-uvloop; platform_system != "Windows"
+uvloop; platform_system != "Windows" and python_version >= '3.5'
```

Comment:
why is it 3.5?

Reply:
Because that’s [their minimum Python version](https://github.com/MagicStack/uvloop#installation). Otherwise, the new Python 2.7 jobs tries to install it, and fails.